defaults/main.yml

```diff
@@ -13,3 +13,9 @@ RABBITMQ_KEY_FILE: "/etc/letsencrypt/live/{{ RABBITMQ_HOSTNAME }}/privkey.pem"
 RABBITMQ_APT_PACKAGES:
   - python-software-properties
   - python-letsencrypt
+
+RABBITMQ_EXPORT_DIR: /usr/local/lib/rabbitmq/backup
```

Comment:
I think if we'd follow the FHS strictly, this would have to go in `/var/cache`, but `/var/local` is fine with me.